openlibrary/templates/type/list/view.html

```diff
@@ -168,14 +168,13 @@ <h1>$list.name</h1>
             $:format(list.description)
 
         <ul id="listResults" class="list-books">
-          $for seed in list.get_seeds(sort=True)[page*page_size:page*page_size+page_size]:
+          $ seeds = list.get_seeds(sort=True)[page*page_size:page*page_size+page_size]
+          $ books = get_book_availability(seeds)
```

Comment:
This is where the lookup table is called from the list front-end (as we iterate over the seeds, when we want to book's corresponding `page` -- i.e. object / doc) we use `books` to fetch it, which contains the availability attribute / dict.